homeassistant/components/rainbird/binary_sensor.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Rain Bird Irrigation system LNK WiFi Module."""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_MONITORED_CONDITIONS
+from homeassistant.helpers.entity import Entity
+from pyrainbird import RainbirdController
+from . import DATA_RAINBIRD
+
+_LOGGER = logging.getLogger(__name__)
+
+# sensor_type [ description, unit, icon ]
+SENSOR_TYPES = {"rainsensor": ["Rainsensor", None, "mdi:water"]}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)): vol.All(
```

Comment:
We don't allow monitored conditions in config anymore.

https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

Reply:
there is single api request for every rainbird sensor state.
Not all are implemented yet, but according the spec CONF_MONITORED_CONDITIONS should be used in this case

Reply:
It's allowed to have monitored conditions if the amount of requests is a problem. We still prefer to not have this option if the integration and api can cope.